pt-BR/mail/chrome/messenger/defaultClientDialog.dtd

```diff
@@ -1,13 +1,13 @@
-<!ENTITY defaultClient.title  "Default Client">
-<!ENTITY defaultClient.intro  "Use &brandShortName; as the default client for:">
+<!ENTITY defaultClient.title  "Cliente padrão">
+<!ENTITY defaultClient.intro  "Usar o &brandShortName; como cliente padrão para:">
 
-<!ENTITY email.label "E-Mail">
-<!ENTITY newsgroups.label "Newsgroups">
-<!ENTITY rss.label "RSS Feeds">
+<!ENTITY email.label "Emails">
```

Comment:
@brunotiussi Understood. After receiving Sherman's approval, either use e-mail or email (chose one to keep consistency).